### PR TITLE
enh: Improve error messages

### LINKED ISSUE #27: Improve message "Failed to close pull request"
Hello,

Currently when the check is not passing we have only the message: "Failed to close pull request".

But it could be due to several reason.

### PR DESCRIPTION
**Pull request by smutel**
Fix #27

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -218,7 +218,8 @@ func main() {
 		)
 
 		if err != nil {
-			log.Fatalln("Failed to close pull request")
+			log.Printf(reason)
```
Comment (Namchee) [inline main.go:221]: As it's generally harder to find unformatted code, could you please format it with `Reason: %s\n`?

**Review by Namchee — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (smutel) [reply main.go:221]: Done.

File: main.go [hunk 2]
```diff
@@ -206,6 +206,16 @@ func main() {
 		reason = fmt.Sprintf("Commit message `%s` at commit %s doesn't follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) style.", badCommit.message, badCommit.url)
 	}
 
+	log.Printf("Title is valid: %s", strconv.FormatBool(isTitleValid))
+	log.Printf("Length of body: %d", len(body))
+	log.Printf("Count of issues: %d", len(issueMentions))
+	log.Printf("Too may changes in PR: %s", strconv.FormatBool(hasTooManyChanges))
```
Comment (Namchee) [inline main.go:212]: Typo here, should be `many` if I interpreted correctly.

**Review by Namchee — CHANGES_REQUESTED** [review]

File: main.go (hunk 2 above)
Comment (smutel) [reply main.go]: Done

**Review by Namchee — APPROVED** [review]
LGTM :100:

Whoops, I'm just gonna bypass the PR for now

**Pull request merged**